### DISCUSSION TITLE
Discussions page for repo

### DISCUSSION THREAD

**Discussion by Alizter** [HoTT/HoTT#1554] [Ideas]
There is a dedicated "discussions" feature which we can enable for the repo. It seems to act like a forum. It might be worth enabling since we have many discussions that are currently issues. This way, there is a clear separation between what needs to be discussed and what needs to be fixed.

Also, keeping discussions as issues is a bit confusing. What does it mean to "close" a discussion?

I think this would be a good idea, so I am opening a discussion to discuss it.

**Comment by Alizter** [comment]
Here is an example:
https://github.com/github/feedback/discussions

**Comment by mikeshulman** [comment]
I think I know what it means to close a discussion, but I can see the value in separating more open-ended discussions from specific issues-to-fix.

**Comment by Alizter** [comment]
It should also be noted that it is very easy to move issues to discussions.

**Comment by mikeshulman** [comment]
i presume we get email notifications for discussions as well as issues?

**Comment by Alizter** [comment]
It seems so. If you like I can enable discussions and we can do a trial. If we don't like it we can always turn it off.

**Comment by jdchristensen** [comment]
I'd like to try it.  I often feel like discussing something that doesn't really fit into the "issue" category.

**Comment by mikeshulman** [comment]
Sure, go ahead.

**Comment by Alizter** [comment]
OK I have enabled it.

**Comment by Alizter** [comment]
As a test, shall we convert this thread to a discussion? The conversion button has now appeared.

**Comment by Alizter** [comment]
So the original issue has been closed and locked by the migration. But we now have a copy in the discussions page.
